common/src/main/java/org/jboss/pnc/bacon/common/cli/JSONCommandHandler.java

```diff
@@ -19,9 +19,14 @@
      */
     protected boolean getJsonOutput() {
         CommandLine.Model.OptionSpec option = commandSpec.root().findOption(Constant.JSON_OUTPUT);
-        if (option == null) {
+        CommandLine.Model.OptionSpec optionFull = commandSpec.root().findOption(Constant.JSON_FULL_OUTPUT);
+        if (option == null && optionFull == null) {
             throw new FatalException("Unable to locate JSON parameter in root");
         }
+        if (option == null) {
+            log.debug("JSON command is enabled: " + optionFull.getValue());
+            return optionFull.getValue();
+        }
         log.debug("JSON command is enabled: " + option.getValue());
```

Comment:
I don't think this is needed. We only need to find the option by name and check if it is enabled.

Reply:
and you're right :+1: